src/EdFi.AnalyticsMiddleTier.Tests/TestHarness.cs

```diff
@@ -229,7 +229,7 @@ public void ExecuteQuery(string sqlCommand)
 
         public void PrepareDatabase()
         {
-            using (var connection = new SqlConnection("server=localhost;database=master;integrated security=sspi"))
+            using (var connection = new SqlConnection("server=localhost;database={_databaseName};integrated security=sspi"))
```

Comment:
Please revert this change.  The PrepareDatabase connection string should use database = master.